src/styles/blocks/main/main.scss

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+
+  .catalog {
```

Comment:
filename should match BEM block name
should be outside of `body` tag

Reply:
thanks for all sugestions, also i have a question about the nesting correctness in src/main/page.scss file. could you give me advise about that please? is it right way to do this?

Reply:
I remember, that you said about re-request, but I'm interested in the thing above and would like to know the answer